app/parsers/gdomain.py

```diff
@@ -0,0 +1,63 @@
+from app.objects.c_relationship import Relationship
+from plugins.stockpile.app.parsers.base_parser import BaseParser
+from app.utility.logger import Logger
+
+class Parser(BaseParser):
+
```

Comment:
Drop this file - It's part of your other PR (https://github.com/mitre/stockpile/pull/209) and we can mark that as a prerequisite/dependency